hw_06/models.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+from pydantic import BaseModel, Field
+
+
+class User(BaseModel):
+    id: int
+    firstname: str = Field(max_length=64)
+    lastname: str = Field(max_length=64)
+    email: str = Field(max_length=64)
+    password: str = Field(min_length=5, max_length=64)
+
+
+class UserIn(BaseModel):
+    firstname: str = Field(None, max_length=64)
+    lastname: str = Field(None, max_length=64)
+    email: str = Field(..., max_length=64)
+    password: str = Field(..., min_length=5, max_length=64)
```

Comment:
Модели очень похожи, а значит можно создать одну из них, наследуясь от другой.
```suggestion
class UserIn(BaseModel):
    firstname: str = Field(None, max_length=64)
    lastname: str = Field(None, max_length=64)
    email: str = Field(..., max_length=64)
    password: str = Field(..., min_length=5, max_length=64)
    

class User(UserIn):
    id: int
```